tools/KoreBuild.Console/Commands/RootCommand.cs

```diff
@@ -22,7 +22,7 @@ public override void Configure(CommandLineApplication application)
             application.Command("upgrade", c =>
             {
                 c.HelpOption("-h|--help");
-                c.Command("deps", new DependenciesUpgradeCommand(context).Configure);
+                c.Command("deps", new DependenciesUpgradeCommand(context).Configure, throwOnUnexpectedArg: false);
```

Comment:
We had to do this to make "upgrade deps" work from our `./test.ps1` script, which passes in some parameters.

Reply:
👍